package.json

```diff
@@ -29,28 +29,24 @@
     "url": "https://github.com/crisbeto/angular-svg-round-progressbar/issues"
   },
   "homepage": "https://github.com/crisbeto/angular-svg-round-progressbar",
-  "peerDependencies": {
-    "@angular/core": "^10.0.0",
-    "@angular/common": "^10.0.0"
-  },
+  "peerDependencies": {},
```

Comment:
Why were the `peerDependencies` removed?

Reply:
I will have a look into this. I can only assume the upgrade removed them

Reply:
I think it should be reverted. I've based this on how it's being done by Angular Material.

Reply:
I have added back missing peer dependencies